### ISSUE TITLE
Couldn't pull from docker [DEPRECATION NOTICE]

### ISSUE THREAD

**Issue by BaDjinn** [fnichol/docker-uhttpd#3]
I've tried to use it but i get:

`[DEPRECATION NOTICE] Docker Image Format v1 and Docker Image manifest version 2, schema 1 support is disabled by default and will be removed in an upcoming release. Suggest the author of docker.io/fnichol/uhttpd:latest to upgrade the image to the OCI Format or Docker Image manifest v2, schema 2. More information at https://docs.docker.com/go/deprecated-image-specs/`

**Comment by webbster64** [comment]
got the same error, did you maybe find a work around or an alternative?

**Comment by BaDjinn** [comment]
Nope, i simply cut it out of the docker-compose where i used it, while i'm waiting for a fix.
